NEWS

```diff
@@ -4,6 +4,15 @@ Please send bug reports, questions and suggestions to
 <https://github.com/besser82/libxcrypt/issues>.
 
 Version 4.3.4
+* Add aliases for xcrypt{,_r} and xcrypt_gensalt{,_r}.
+  They were added for code compatibility with libxcrypt v3.1.1
+  and earlier.
+* Install a symlink from libcrypt.so to libxcrypt.so, if a shared
```

Comment:
Isn't it the other way around? (`libxcrypt.so -> libcrypt.so`)

Reply:
Will fix that in rebased commit.